### PR TITLE
Sf Select

### PR DESCRIPTION
**Pull request by przemyslawspaczek**
# Related issue
#80 

# Scope of work
Created Select Component with styling. 

# Screenshots of visual changes
open: false
SfSelect--close
open: true
SfSelct--open

# Checklist

- [x] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[0513/125946.675:ERROR:crash_report_database_win.cc(428)] unexpected header
```
Comment (Jensderond) [inline debug.log]: Please remove this

**Comment by Jensderond** [comment]
At the moment this happens:
GIF displaying the problem

I think this is not wanted from a UX standpoint.
When you'd quickly click and close the same select window, the selected item changes. I believe the options should be under the already selected item.

**Comment by filrak** [comment]
@Jensderond you mean it changes to itself? Imho we shoudl just check if new item isn't current one @przemyslawspaczek

**Comment by Jensderond** [comment]
@filrak No what I mean is that when you (accidentally) double click on the select box you could select another item then you started with, as shown in the GIF

**Comment by filrak** [comment]
Ah you're right.

**Comment by przemyslawspaczek** [comment]
We can:
A: move options under selected item
selectA
B: delay click event, then when you click to fast, you do nothing 
C: sort list and always first item will be same as selected item

**Comment by Jensderond** [comment]
@przemyslawspaczek I think option A is the most user-friendly

**Comment by filrak** [comment]
@przemyslawspaczek please solve conflicts and I can merge! ;)

**Comment by przemyslawspaczek** [comment]
@filrak done

**Pull request merged**